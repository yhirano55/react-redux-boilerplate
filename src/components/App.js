import React from 'react';
import CSSModules from 'react-css-modules';
import styles from '../stylesheets/App.css';

const App = (props) => (
  <p styleName="title">Hello, {props.name}</p>
)

export default CSSModules(App, styles);
