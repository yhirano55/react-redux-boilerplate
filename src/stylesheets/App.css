.title {
  color: #333;
  font-size: 30px;
  font-weight: 700;
}
