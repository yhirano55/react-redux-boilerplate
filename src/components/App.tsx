import * as React from 'react';
import * as styles from '../stylesheets/App.css';

export default class App extends React.Component {
  render(): JSX.Element {
    return (
      <p className={styles.title}>Hello, World</p>
    );
  }
}
