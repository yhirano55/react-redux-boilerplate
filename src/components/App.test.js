import React from 'react'
import renderer from 'react-test-renderer'

import App from './App'

describe('<App />', () => {
  context('with normal scenario', () => {
    const component = renderer.create(
      <App />
    )

    it('should display App', () => {
      const tree = component.toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
