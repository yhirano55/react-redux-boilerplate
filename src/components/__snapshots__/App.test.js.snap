// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> with normal scenario should display App 1`] = `
<p>
  Hello, World
</p>
`;
